ui/packages/consul-hcp/app/components/consul/hcp/home/index.hbs

```diff
@@ -2,7 +2,7 @@
   class="consul-hcp-home"
   ...attributes
 >
-  <a href={{env 'CONSUL_HCP_URL'}}>
+  <a href={{env 'CONSUL_HCP_URL'}} data-native-href="true">
```

Comment:
Why are we adding this data-attribute here? Is this changing the behavior of the link or its styling? I don't see it being used anywhere else in the application. Is this a left-over from development and can be removed?

Reply:
It disables the helper from the ember href-to addon from preventingDefault and dealing with the event itself, so no, we do need this right now

Reply:
I might be missing something here, but aren't we implementing the `href-to` ourselves? I don't see where we are including/extending the ember-href-to-addon?!